### ISSUE TITLE
Failed to import games from Battle.net.

### ISSUE THREAD

**Issue by Lombra** [JosefNemec/Playnite#868]
Just found out about and installed this the other day. Looks promising! However, I did not come far. When trying to update Battle.net games I get the following error, and no games get imported:
```
Failed to import games from Battle.net.
An item with the same key has already been added.
```
Here is the diagnostics ID I got:
8b22107b-47f5-4fd7-b8d2-306f86cbe695

I'm using 4.63, which I noticed curiously has an entry for fixing Battle.net importing.

**Comment by JosefNemec** [comment]
Do at least installed games import properly if you disable import of uninstalled ones?

**Comment by Lombra** [comment]
Nope. Don't have uninstalled games checked. It does say Library update finished in the title bar, in case that matters.

**Comment by JosefNemec** [comment]
Thanks for more info. I know where the issue is now and there will update to fix this released next week.

**Comment by JosefNemec** [comment]
Fixed in 4.64
